packages/base/room.gts

```diff
@@ -576,26 +576,30 @@ export class RoomField extends FieldDef {
             ...cardArgs,
             attachedCardIds,
           });
-        } else if (event.content.msgtype === 'org.boxel.command') {
+        } else if (
+          event.content.msgtype === 'org.boxel.command' &&
+          event.content.data.toolCall
+        ) {
           // We only handle patches for now
-          let command = event.content.data.command;
+          let commandEvent = event as CommandEvent;
+          let command = event.content.data.toolCall;
           let annotation = this.events.find(
             (e) =>
               e.type === 'm.reaction' &&
               e.content['m.relates_to']?.rel_type === 'm.annotation' &&
               e.content['m.relates_to']?.event_id ===
                 // If the message is a replacement message, eventId in command payload will be undefined.
                 // Because it will not refer to any other events, so we can use event_id of the message itself.
-                (command.eventId ?? event_id),
+                (commandEvent.content.data.eventId ?? event_id),
           ) as ReactionEvent | undefined;
 
           messageField = new MessageField({
             ...cardArgs,
-            formattedMessage: `<p class="patch-message">${event.content.formatted_body}</p>`,
-            command: new PatchField({
+            formattedMessage: `<p class="command-message">${event.content.formatted_body}</p>`,
+            command: new CommandField({
               eventId: event_id,
-              commandType: command.type,
-              payload: command,
+              commandType: command.name,
+              payload: command.arguments,
```

Comment:
should we change commandType to name and payload to arguments here too? My initial naming wasn't great.

Reply:
I am in favor of this

Reply:
```
Code contained in a class is evaluated in JavaScript's strict mode which does not allow this use of 'arguments'. For more information, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode.
```

It looks like arguments is protected in strict mode

Reply:
Urgh. I like having the same name across things though, happy with any alternative name that doesn't cause an issue.

Reply:
I was considering `args` but I think payload is actually ok for now